mesop/web/src/editor/editor.ts

```diff
@@ -163,7 +165,7 @@ class Editor {
   }
 
   showEditorToolbar(): boolean {
-    return Boolean(window.localStorage.getItem('MESOP://SHOW_EDITOR_TOOLBAR'));
+    return this.experimentService.experimentalEditorToolbarEnabled;
```

Comment:
Nice. Does this mean we don't have to enable the visual editor in Chrome dev tools anymore along with the env var?

Reply:
yup!